### PR TITLE
perf/perf_pmu: Extend the test to support Power11

### PR DESCRIPTION
**Pull request by sacsant**
Extend the test case to support Power11 processor

Signed-off-by: Sachin Sant <sachinp@linux.ibm.com>

### CODE REVIEW

**Review by disgoel — APPROVED** [review]
LGTM

File: perf/perf_pmu.py [hunk 1]
```diff
@@ -112,7 +112,8 @@ def test_config_PMU_sysfs(self):
         sysfs_dict = {"power8": ['mmcr0', 'mmcr1', 'mmcra'],
                       "power8e": ['mmcr0', 'mmcr1', 'mmcra'],
                       "power9": ['mmcr0', 'mmcr1', 'mmcra'],
-                      "power10": ['mmcr0', 'mmcr1', 'mmcr3', 'mmcra']}
+                      "power10": ['mmcr0', 'mmcr1', 'mmcr3', 'mmcra'],
+                      "power11": ['mmcr0', 'mmcr1', 'mmcr3', 'mmcra']}
```
Comment (PraveenPenguin) [inline perf/perf_pmu.py:116]: I think @disgoel we spoke last time as well, not sure how hard this to drive from YAML as code changes is always cost maintenance

Comment (Naresh-ibm) [reply perf/perf_pmu.py:116]: I agree with @PraveenPenguin . Better to pass from Yaml or use a config file.

Comment (sacsant) [reply perf/perf_pmu.py]: Have introduced a cfg file and introduced configparser to read the data from it.

**Comment by sacsant** [comment]
Before the change:
 (1/8) perf/perf_pmu.py:PerfBasic.test_config_PMU_sysfs: STARTED
 (1/8) perf/perf_pmu.py:PerfBasic.test_config_PMU_sysfs:  ERROR: 'power11' (0.06 s)
 (2/8) perf/perf_pmu.py:PerfBasic.test_config_PPC_RTAS: STARTED
 (2/8) perf/perf_pmu.py:PerfBasic.test_config_PPC_RTAS:  PASS (0.54 s)
 (3/8) perf/perf_pmu.py:PerfBasic.test_cpu_event_count: STARTED
 (3/8) perf/perf_pmu.py:PerfBasic.test_cpu_event_count:  PASS (0.03 s)
 (4/8) perf/perf_pmu.py:PerfBasic.test_hv_24x7_event_count: STARTED
 (4/8) perf/perf_pmu.py:PerfBasic.test_hv_24x7_event_count:  PASS (0.03 s)
 (5/8) perf/perf_pmu.py:PerfBasic.test_imc_event_count: STARTED
 (5/8) perf/perf_pmu.py:PerfBasic.test_imc_event_count:  SKIP: This test is for PowerNV
 (6/8) perf/perf_pmu.py:PerfBasic.test_thread_event_count: STARTED
 (6/8) perf/perf_pmu.py:PerfBasic.test_thread_event_count:  SKIP: This test is for PowerNV
 (7/8) perf/perf_pmu.py:PerfBasic.test_nest_cpumask: STARTED
 (7/8) perf/perf_pmu.py:PerfBasic.test_nest_cpumask:  SKIP: This test is for PowerNV
 (8/8) perf/perf_pmu.py:PerfBasic.test_core_cpumask: STARTED
 (8/8) perf/perf_pmu.py:PerfBasic.test_core_cpumask:  SKIP: This test is for PowerNV

After the change:
 (1/8) perf/perf_pmu.py:PerfBasic.test_config_PMU_sysfs: STARTED
 (1/8) perf/perf_pmu.py:PerfBasic.test_config_PMU_sysfs:  PASS (0.37 s)
 (2/8) perf/perf_pmu.py:PerfBasic.test_config_PPC_RTAS: STARTED
 (2/8) perf/perf_pmu.py:PerfBasic.test_config_PPC_RTAS:  PASS (0.31 s)
 (3/8) perf/perf_pmu.py:PerfBasic.test_cpu_event_count: STARTED
 (3/8) perf/perf_pmu.py:PerfBasic.test_cpu_event_count:  PASS (0.03 s)
 (4/8) perf/perf_pmu.py:PerfBasic.test_hv_24x7_event_count: STARTED
 (4/8) perf/perf_pmu.py:PerfBasic.test_hv_24x7_event_count:  PASS (0.03 s)
 (5/8) perf/perf_pmu.py:PerfBasic.test_imc_event_count: STARTED
 (5/8) perf/perf_pmu.py:PerfBasic.test_imc_event_count:  SKIP: This test is for PowerNV
 (6/8) perf/perf_pmu.py:PerfBasic.test_thread_event_count: STARTED
 (6/8) perf/perf_pmu.py:PerfBasic.test_thread_event_count:  SKIP: This test is for PowerNV
 (7/8) perf/perf_pmu.py:PerfBasic.test_nest_cpumask: STARTED
 (7/8) perf/perf_pmu.py:PerfBasic.test_nest_cpumask:  SKIP: This test is for PowerNV
 (8/8) perf/perf_pmu.py:PerfBasic.test_core_cpumask: STARTED
 (8/8) perf/perf_pmu.py:PerfBasic.test_core_cpumask:  SKIP: This test is for PowerNV

**Review by PraveenPenguin — APPROVED** [review]

**Pull request merged**